zipline/src/jniTest/kotlin/app/cash/zipline/QuickJsInboundChannelJvmTest.kt

```diff
@@ -68,8 +70,7 @@ class QuickJsInboundChannelJvmTest {
     val t = assertFailsWith<QuickJsException> {
       inboundChannel.disconnect("service one")
     }
-    assertThat(t)
-      .hasMessageThat()
+    assertThat(t.message!!)
```

Comment:
There is an equivalent to `hasMessageThat()` in AssertK. I think it's just `message()`. We can clean them all up afterwards though.

Reply:
👍🏻